Docs/General/BasicGit.md

```diff
@@ -11,9 +11,14 @@ This means that while one person works on code for the autonomous, another could
 
 ### Resources
 
+Read one of the following:
+
 - [WPILib Git intro](https://docs.wpilib.org/en/stable/docs/software/basic-programming/git-getting-started.html)
 - [Github Git intro](https://docs.github.com/en/get-started/using-git/about-git)
-- _Record quick intro to git vid/talk at some point, talk to kevin or a software lead until then_
+- _Record quick intro to git vid/talk at some point, talk to kevin or a software lead until then if you prefer a non-text intro_
+
+Install the following:
+
 - [Git install](https://gitforwindows.org/)
```

Comment:
ik macs usually come with git but would it be helpful to include a link for that also